File: assignment1/fourargs.cpp
#include <iostream>
int main(int argumentcount, char** arguments)
{
if(argumentcount>4){
	for(int i=1;i<5;i++){
		std::fprintf(stdout,"%s\n",arguments[i]);
	}
	for(int j=5;j<argumentcount;j++){
		std::fprintf(stderr,"%s\n",arguments[j]);
	}
}
else{
	for(int k=1;k<argumentcount;k++){
		std::fprintf(stdout,"%s\n",arguments[k]);
	}
}
	return 0;
}

// Copyright 2017 Zhonghao Guo gzh1994@bu.edu

File: assignment1/fourargs.py
# Copyright 2017 Zhonghao Guo gzh1994@bu.edu
import sys
if(len(sys.argv)>5):
 for i in range(1,5):sys.stdout.write(sys.argv[i]+'\n')

 for j in range(5,len(sys.argv)):sys.stderr.write(sys.argv[j]+'\n')


else:
 for k in range(1,len(sys.argv)):sys.stdout.write(sys.argv[k]+'\n')

 
